src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  user: User | null,
+  todo: Todo | null,
+  onCloseBtn: ()=>void,
```

Comment:
please pay attention to spaces
```suggestion
  onCloseBtn: () => void,
```